frontend/src/views/ElectedCanteens.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="text-left">
```

Comment:
I think that the ElectedCanteens page could be the home page for elected officials

Reply:
Yeah but I would be more comfortable making that change a bit later down the road, once there's a bit more functionalities to it.